plugin/cmd/main.go

```diff
@@ -185,7 +183,15 @@ func plugin(ctx *cli.Context) {
 
 		avsManager.RegisterOperatorWithAvs(ethHttpClient, operatorEcdsaPrivateKey, blsKeypair)
 	} else if operationType == "opt-out" {
-		fmt.Println("Opting out of slashing - unimplemented")
+		blsKeyPassword := ctx.GlobalString(BlsKeyPasswordFlag.Name)
+
+		blsKeypair, err := bls.ReadPrivateKeyFromFile(avsConfig.BlsPrivateKeyStorePath, blsKeyPassword)
+		if err != nil {
+			logger.Error("Failed to read bls private key", "err", err)
+			return
+		}
+
+		avsManager.DeregisterOperator(blsKeypair)
```

Comment:
Error is never propagated

Reply:
Added this feature: we now propagate all error values from `plugin`